test/subproc-specs.js

```diff
@@ -173,21 +183,17 @@ describe('SubProcess', function () {
             resolve();
           }
         });
-        await subproc.start();
-      }).should.eventually.not.be.rejected;
+        return subproc.start();
+      })).should.eventually.not.be.rejected;
     });
     it('should get output as params', async function () {
-      await new B(async (resolve, reject) => {
         subproc = new SubProcess(getFixture('echo'), ['foo', 'bar']);
         subproc.on('output', (stdout, stderr) => {
           if (stderr && stderr.indexOf('bar') === -1) {
-            reject();
-          } else {
-            resolve();
+            throw new Error();
```

Comment:
I believe this change would create an uncaught exception failure

Reply:
I tried flipping the `if` statement and  it stops the test, so the exception doesn't hang